stack/packaging-third-party-eups-dependencies.rst

```diff
@@ -46,7 +46,9 @@ The upstream source has no need to know about the ``ups`` directory, so we put i
 That branch is traditionally named ``lsst-dev``.
 We set that branch to be the GitHub default in our fork.
 
-When changes are made to the upstream source, they can be merged into the ``lsst-dev`` branch, or ``lsst-dev`` can be rebased on an upstream branch.
+The ``main`` or the ``master`` branch of the fork should track the default branch upstream, and can be synced as often as desired.
+When relevant changes are made to the upstream source, they can be merged into the ``lsst-dev`` branch, or ``lsst-dev`` can be rebased on an upstream branch.
```

Comment:
I'm trying to understand what we were trying to say here (and it's in the original) since won't rebasing lsst-dev mean that we detach all the w.YYYY.NN tags from the history?

Reply:
OK, my guess here is that rebasing is fine (and may even be preferred) for repos that don't receive weekly tags (metadetect is one such example). Once they are part of `lsst_distrib` or other distributions, they should also get a merge from a ticket branch.